### PR TITLE
Create mirror.oit.ncsu.edu  for NCSU public repository

### PR DESCRIPTION
**Pull request by thebarbariccrayon**

### CODE REVIEW

**Comment by jonathanspw** [comment]
Thanks for setting up a mirror.

Unfortunately the URLs you listed do not load.  Is this intended to be a private mirror or is something wrong?

**Comment by jonathanspw** [comment]
Please let us know when it is ready to go and then we can merge it.

**Comment by thebarbariccrayon** [comment]
Mirrors are up and ready.

**Comment by codyro** [comment]
Hi @thebarbariccrayon,

It looks like the mirror isn't updating, would you mind looking into this before we merge it?

> Wed Jun 19 18:14:49 UTC 2024

Ref; 

https://mirror.linux.ncsu.edu/almalinux/timestamp.txt

Thanks :)

**Comment by thebarbariccrayon** [comment]
Sure!

**Comment by thebarbariccrayon** [comment]
Mirror has been updated.

**Comment by jonathanspw** [comment]
For many college campuses listing an ASN can be beneficial to have all of your internal traffic hit your own mirror.  Is that something you'd like to do?

In the end you know your network so it may or may not be beneficial for you and is up to you.

**Comment by thebarbariccrayon** [comment]
ASN: 11442

File: mirrors.d/mirror.oit.ncsu.edu.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  https: https://mirror.linux.ncsu.edu/almalinux/
+  rsync: rsync://mirror.linux.ncsu.edu/almalinux/
+update_frequency: 15m
+geolocation:
+  country: US
+  state_province: North Carolina
+  city: Raleigh
+sponsor: North Carolina State University
+sponsor_url: https://linux.ncsu.edu
+email: group-oit-csi@ncsu.edu
```
Comment (jonathanspw) [inline mirrors.d/mirror.oit.ncsu.edu.yml:14]:
```suggestion
email: group-oit-csi@ncsu.edu
asn: 11442
```

**Comment by jonathanspw** [comment]
That's for hosting a mirror!

**Pull request merged**